File: sunemedicpro_inventarios.client/src/app/dtos/inventarios/estudianteDTO.ts
export interface EstudianteDTO {
  id: number;
  documento: string;
  nombres: string;
  apellidos: string;
  fechaRegistro: Date;
  usuarioId: number;
}


export interface EstudianteCreacionDTO {
  documento: string;
  nombres: string;
  apellidos: string;
}
export interface EstudianteUpdateDTO {
  documento: string; 
  nombres: string;   
  apellidos: string;
}

File: RegistroEstudiantil.Application/DTOs/EstudianteDTO.cs
namespace RegistroEstudiantil.Application.DTOs
{
    public class EstudianteDTO
    {
        public int Id { get; set; }
        public string Documento { get; set; } = null!;
        public string Nombres { get; set; } = null!;
        public string Apellidos { get; set; } = null!;
        public DateTime FechaRegistro { get; set; } = DateTime.UtcNow;

        // Relación 1-1 con Usuario para el “registro en línea”
        public int UsuarioId { get; set; }
    }
}


